src/components/GetImageButton.js

```diff
@@ -1,24 +1,11 @@
 import React, {Component} from 'react';
 
-const API_KEY = 'X3aNQZj7BdSiNk8EUjNBImIKplVeWrVEfeVislMQ';
-
-export default class GetImageButton extends Component{
-
-fetchRoverImage(){
-
-  this.setState({camera: this.props.camera, rovers: this.props.rovers, sol: this.props.sol});
```

Comment:
Stateless components, as the name suggest, don't have state, so there's no such thing as `this.setState`.